.evergreen/install-node-source.sh

```diff
@@ -7,6 +7,16 @@ EVGDIR="$ROOT_DIR/.evergreen"
 NVM_DIR="$EVGDIR/.nvm"
 ORIGINAL_PATH="${PATH}"
 
+OS_ARCH="$(uname "-m")"
+if [ "$OS_ARCH" = "ppc64le" ] || [ "$OS_ARCH" = "ppc64" ] ; then
+    echo "[INFO] Choosing v4 because OS_ARCH is $OS_ARCH"
```

Comment:
Just a small note, in scripts that already have `set -x` set, echoing back debug information can be nice for formatting but doesn't usually provide other benefits, since bash will already echo back the values for OS_ARCH, PATH, etc. when they are being set

Reply:
Yeah, I guess there's a small benefit of making this easier to grep if you notice it in the parsley logs + that was the original code before it got removed in https://github.com/mongodb-js/mongosh/commit/315fefa444e36c982a8e6905976199aaf2f5793d, so figured might keep it like that. I don't have strong opinions though and happy to remove it to keep the script more concise.